### ISSUE TITLE
🐛Double quotes in HTML attributes are not supported

### ISSUE THREAD

**Issue by bfred-it** [parcel-bundler/parcel#988]
**Choose one:** is this a 🐛 bug report or 🙋 feature request?

🐛Bug

Double quotes in HTML attributes are flat-out not supported, not even when escaped or when inside single quotes

Both `<html attr='"'>` and `<html attr="&quot;">` come out as `<html attr="" "="">`

### 🎛 Configuration (.babelrc, package.json, cli command)

No config

```html
<!DOCTYPE html><body aria-label='this is a "bug"'></body>
```

```sh
parcel build index.html
```

### 🤔 Expected Behavior

Output should be unchanged:

```html
<!DOCTYPE html><body aria-label='this is a "bug"'></body>
```

### 😯 Current Behavior

Single quotes are changed to double quotes, without escaping the double quotes inside the attribute, therefore breaking the HTML.

```html
<!DOCTYPE html><body aria-label="this is a " bug""=""></body>
```

### 🔦 Context

I'm trying to use JSON in the attribute, as suggested for [example by Flickity](https://flickity.metafizzy.co/#initialize-with-html)

### 💻 Code Sample

```html
<!DOCTYPE html><body aria-label='this is a "bug"'></body>
```

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.6.2
| Node             | 5.6.0
| npm/Yarn         | 5.7.1
| Operating System | macOS 10.13.3

**Comment by bfred-it** [comment]
I think this is actually a duplicate of https://github.com/parcel-bundler/parcel/issues/912, but I don't think it's resolved. The HTML pasted there isn't actually the HTML in `dist` but what the browser eventually parses and re-displays as HTML in the inspector.

**Comment by DeMoorJasper** [comment]
It might be related, it never got fixed indeed. But it appears to be unrelated to parcel and a htmlnano or post-html issue

**Comment by bfred-it** [comment]
Is there a way to disable **all** HTML changes? I'm seeing one more bug during the minification and I don't want to disable minification on the whole project

**Comment by bfred-it** [comment]
Apparently addressed by #22 9 days ago
